apache-Dockerfile-block-2

```diff
@@ -1,5 +1,50 @@
 COPY apache2-foreground /usr/local/bin/
 WORKDIR /var/www/html
 
+# Set up FPM
+RUN set -ex \
+	&& cd /usr/local/etc \
+	&& if [ -d php-fpm.d ]; then \
+		# for some reason, upstream's php-fpm.conf.default has "include=NONE/etc/php-fpm.d/*.conf"
+		sed 's!=NONE/!=!g' php-fpm.conf.default | tee php-fpm.conf > /dev/null; \
```

Comment:
`| tee php-fpm.conf > /dev/null` seems super pointless, just redirect to `> php-fpm.conf` directly?

Reply:
Hi! These lines were copied directly from the [current FPM dockerfile](https://github.com/docker-library/php/blob/master/fpm-Dockerfile-block-2#L7).
I think these corrections should be done there first, in order to not maintain different versions of the snippet.